DEPENDENCIES.txt

```diff
@@ -32,6 +32,7 @@ FindBin
 Getopt::Long
 IO::Socket::INET6
 IO::Socket::SSL
+IPC::Run::Debug
```

Comment:
os-autoinst has *IPC::Run::Debug* in it's DEPENDENCIES.txt so I wanted to make sure that I use the same RPM.

But I guess the rpm should also provide IPC::Run - do you want me to change it?